hs_app_netCDF/models.py

```diff
@@ -349,4 +263,10 @@ def add_metadata_element_to_xml(self, root, md_element, md_fields):
                                                                       field=md_fields[md_field]))
                     field.text = str(attr)
 
+    def delete_all_elements(self):
+        super(NetcdfMetaData, self).delete_all_elements()
+        if self.ori_coverage.all().first():
```

Comment:
I think this should be delete all and not just the first record. (```self.ori_coverage.all().delete()```). No need of the if statement.